chatcommunicate.py

```diff
@@ -269,12 +306,19 @@ def handle_commands(content_lower, message_parts, ev_room, ev_room_name, ev_user
             t_metasmoke.start()
 
             add_ignored_post(post_site_id[0:2])
-            if not quiet_action:
-                return "Recorded answer as an NAA in metasmoke."
-            else:
+            if quiet_action:
                 return None
-        if (second_part_lower.startswith("delete") or second_part_lower.startswith("remove") or second_part_lower.startswith("gone") or second_part_lower.startswith("poof") or
-                second_part_lower == "del") and is_privileged(ev_room, ev_user_id, wrap2):
+
+            return "Recorded answer as an NAA in metasmoke."
+
+        if (
```

Comment:
Yeah... flake8 was really digging into me there if I changed it from a oneliner <_<